lib/davinci_dtr_test_kit/docs/dtr_light_ehr_suite_description_v201.md

```diff
@@ -18,12 +18,21 @@ Once the connection between the DTR SMART App and the DTR Light EHR is establish
 
 ## Running the Tests
 
-If you would like to try out the tests but don't have a DTR payer server implementation, you can run these tests against the [public instance of the Inferno Reference Server](https://inferno.healthit.gov/reference-server/r4/) by using the Inferno Reference Server preset in the test suite.
+If you would like to try out the tests but don't have a DTR Light EHR implementation, you can run these tests against the [public instance of the Inferno Reference Server](https://inferno.healthit.gov/reference-server/r4/) by using the Inferno Reference Server preset in the test suite.
 
-In order to get the Inferno QA Reference Server to do an EHR launch, navigate to https://inferno.healthit.gov/reference-server/app/app-launch and use https://inferno.healthit.gov/custom/smart/launch as the App Launch URL.
+In order to get the Inferno Reference Server to do an EHR launch, navigate to https://inferno.healthit.gov/reference-server/app/app-launch and use https://inferno.healthit.gov/custom/smart/launch as the App Launch URL.
 
```

Comment:
Let's add some expectation on what can be tested and what will pass vs fail. Specifically, highlight that
- create / update tests are not tested
- The sample patient is focused on DTR-specific data and does not have the complete set of US Core data needed to pass all the US Core tests

Reply:
Addressed in [Suite description changes](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/dcec5709c38a65552ae01ee64edfa0be090dcddc)

Additionally, two questions:
1. Should I add something about the Inferno Reference Server (Local) preset and expectations for that?
2. Should I create a patient that contains DTR-specific data AND has a complete set of US Core data? (I know we are in a time crunch for this PR so if the answer is not enough time, perhaps I could create a backlog task for this if desired?)

Reply:
1. No, because we won't publish that local preset. This could go into the README for developers
2. Let's put a ticket on the backlog. I'm not sure its a high priority. I also actually think that we are likely to move away from testers provide specific ids and instead perform a SMART launch to provide those details, but not for this initial release.

Reply:
I think this conversation can be resolved - I don't have any additional changes needed.